safe/definitions/utilities.py

```diff
@@ -1,6 +1,6 @@
 # coding=utf-8
 """Utilities module for helping definitions retrieval."""
-
+import os
 from os.path import join, exists
```

Comment:
It's only a small comment. We were already importing `from os.path import join,exists` you could just add `splittext` on this line as it's the only function you are using. We also try having one blank line after module docstring